README.md

```diff
@@ -35,14 +36,36 @@ This will open a tree view, showing:
     Name>/src` folder and putting a file in it that has the `.lf` extension.
  3. Open a Lingua Franca project (`File > Add Folder to Workspace...`). The Explorer (upper left in sidebar) should now show your project files. Open the `.lf` file you created.
  4. (Optional) Show the diagram for this file by clicking on the diagrams icon at the upper right: ![image](https://user-images.githubusercontent.com/33707478/130875545-ad78a9b7-a07b-4eb9-be59-f6c758cc816b.png))
- 5. Compile the `.lf` file and run it using (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>, then `Lingua Franca: Build and Run`).
+ 5. Compile the `.lf` file and run it using the run button left of the diagrams button or by typing (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>, then `Lingua Franca: Build and Run`).
+
+## Use Local Clone
+
+If you wish to make changes to the VS Code extension or to the `lingua-franca` repository that it uses, you can make a local clone an install that as your VS Code extension:
```

Comment:
Note that the contents of this file are shown as the description of the extension (inside VS Code and on the marketplace). It should describe _what the extension does and why it is useful_, not how expert developers can setup workflows to work on the Lingua Franca code base. I suggest moving the "Use Local Clone" content to the `CONTRIBUTING.md`.

Reply:
Ah, I didn't realize that the README.md file was used as the description in the marketplace.  I've revised it in d48a8b95295bdf78f8202ae58834ac2de10af5bc to just point to the CONTRIBUTING.md file.

There is a test failure that I don't understand at all: https://github.com/lf-lang/vscode-lingua-franca/actions/runs/14553940892/job/40828358833. Suggestions?

Reply:
That test is ensuring that outdated dependencies are detected correctly. Somehow, that's not happening with `pylint`.

Reply:
Why is pylint involved?  This PR is trying to update README files.  Where is Python even involved in this repo?  Clearly we have way too many dependencies. Can we just remove the dependency on pylint altogether?

Reply:
I see that this PR is mostly about updating docs, but sometimes test failures like these are introduced by changes in the configuration of the runners that execute the test -- hence they could be entirely unrelated to changes you made in the code. That being said, there are currently test failures in `main` as well. I don't remember the details, but I'm pretty sure `pylint` is used for Lingua Franca's Python target.